game-core/src/main/java/games/strategy/engine/chat/IStatusChannel.java

```diff
@@ -4,6 +4,9 @@
 import games.strategy.engine.message.RemoteName;
 import games.strategy.net.INode;
 
+/**
+ * A channel through which a chat participant's status is communicated.
```

Comment:
Noting that 'channel' appears in the name of the class. What is a 'channel'?

Reply:
The entire `g.s.engine.message` package centers around the concept of a "channel."  Almost every type refers to a channel, with `IChannelMessenger` probably doing the best job trying to define it.  IMO, the best solution would be to document the channel architecture (including the definition of "channel") at the package level, not in an individual type.